### PR TITLE
feat: Added wishlist management

### PR DESCRIPTION
**Pull request by xii-vi**
### Added Wishlist management
Wishlist-RE-KICKS

### CODE REVIEW

**Comment by nikhil-sharma-nks** [comment]
looks good to be merged, nice layout👍

**Comment by HarshnaNHaswani** [comment]
If the items are in wishlist, all items should have heart icon

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <div class="card-img">
+                <img src="https://res.cloudinary.com/dx8gt3nnu/image/upload/q_auto,f_auto/dryp-shoe-pictures/air-jordan-1-retro-high-og-'electro-orange'-555088180/display_picture.png"
+                    alt="Air Jordan 1 Retro High OG 'Electro Orange'" />
+                <div class="p-3">
+                    <small>Nike</small>
+                    <p class="bold-text">Air Jordan 1 Retro High OG 'Electro Orange'</p>
+                    <small>₹ 30,995</small>
+                </div>
+                <div class="wishlist-icon p-2"><i class="fa fa-heart fa-2x"></i></div>
+                <div class="m-4 ">
```
Comment (HarshnaNHaswani) [inline pages/wishlist.html:70]: use proper class names

**Review by HarshnaNHaswani — APPROVED** [review]
Nice design

File: pages/wishlist.html (hunk 1 above)
Comment (xii-vi) [reply pages/wishlist.html:70]: _Suggested Changes are made._

**Review by saifudinbadani — APPROVED** [review]
looks good to merge.

**Pull request merged**